core/controllers/acl_decorators_test.py

```diff
@@ -2273,6 +2273,19 @@ def test_redirect_with_correct_classroom_name_in_url(self):
                 % self.story_url_fragment,
                 response.headers['location'])
 
+    def test_redirect_lowercase_story_url_fragment(self):
+        topic_services.publish_topic(self.topic_id, self.admin_id)
+        topic_services.publish_story(
+            self.topic_id, self.story_id, self.admin_id)
+        with self.swap(self, 'testapp', self.mock_testapp):
+            response = self.get_html_response(
+                '/mock_story_page/staging/topic/story/Story-frag',
+                expected_status_int=302)
+            self.assertEqual(
+                'http://localhost/learn/staging/topic/story/%s'
+                % self.story_url_fragment,
```

Comment:
Better to just inline "story-frag" here, IMO, similar to what you've done in the other two tests.

Reply:
Addressed this comment in https://github.com/oppia/oppia/pull/10304. Thanks!